src/polarization.jl

```diff
@@ -8,30 +8,63 @@ using ..Convention
 export Polarization0_ZeroTemp, Polarization0_FiniteTemp
 
 srcdir = "."
-rundir = isempty(ARGS) ? pwd() : (pwd()*"/"*ARGS[1])
+rundir = isempty(ARGS) ? pwd() : (pwd() * "/" * ARGS[1])
 
 # using Parameters, GreenFunc, CompositeGrids
 # include(srcdir*"/parameter.jl")
 # using .Parameter
 # include(srcdir*"/convention.jl")
 # using .Convention
 
-# Analytical calculated integrand of Π0.
-@inline function _ΠT_integrand(k, q, ω, param)
-    @unpack me, β, EF, kF = param
+# Analytical calculated integrand of Π0 in 2D.
+# @inline function _ΠT2d_integrand(k, q, ω, param)
+#     @unpack me, β, EF = param
+#     density = me / 2π
+
+#     v1 = me * ω - q^2 / 2
+#     v2 = me * ω + q^2 / 2
+#     nk = 1.0 / (exp(β * (k^2 / 2 / me - EF)) + 1)
+
+#     theta1, theta2 = abs(v1) - k * q, abs(v2) - k * q
+#     if theta1 <= 0 && theta2 <= 0
+#         integrand = 0.0
+#     elseif theta1 <= 0 && theta2 > 0
+#         integrand = k * nk * (-sign(v2) / √(v2^2 - k^2 * q^2))
+#     elseif theta1 > 0 && theta2 <= 0
+#         integrand = k * nk * sign(v1) / √(v1^2 - k^2 * q^2)
+#     else
+#         integrand = k * nk * (sign(v1) / √(v1^2 - k^2 * q^2) - sign(v2) / √(v2^2 - k^2 * q^2))
+#     end
+
+#     return density * integrand
+# end
+
+# Analytical calculated integrand of Π0 in 2D.
+@inline function _ΠT2d_integrand(k, q, ω, param)
+    @unpack me, β, EF = param
+    density = me / 2π
+    nk = 1.0 / (exp(β * (k^2 / 2 / me - EF)) + 1)
+
+    return nothing
+end
+
+# Analytical calculated integrand of Π0 in 3D.
+@inline function _ΠT3d_integrand(k, q, ω, param)
```

Comment:
\omega is an integer or float? If it is an integer, explicitly specify the type.